modules/aleph/src/Aleph/AlephDebt.php

```diff
@@ -85,7 +87,7 @@ public function getSum() {
    *    The cash transaction date.
    */
   public function setDate($date) {
-    $this->date = $date;
+    $this->date = DateTime::createFromFormat('Ymd', $date)->format('Y-m-d');
```

Comment:
I'd prefer it if the format was documented in the docblock, I know that the setter is primarily used by the factory, but some day it might be used from somewhere.